pkg/pillar/docs/zedkube.md

```diff
@@ -107,8 +106,8 @@ NodeDrainStatus can be forced by writing the object (in pillar svc container fs)
 eg. to force disable drain:
 echo '{"Status":1,"RequestedBy":1}' > /tmp/force-NodeDrainStatus-global.json
 
-eg. to force drain complete:
-echo '{"Status":1,"RequestedBy":1}' > /tmp/force-NodeDrainStatus-global.json
+eg. to force deviceop drain complete:
+echo '{"Status":9,"RequestedBy":2}' > /tmp/force-NodeDrainStatus-global.json
```

Comment:
In other file you changed this path to /persist ? You need to update in this file.